### PR TITLE
Data Disk Duplicator

### PR DESCRIPTION
**Pull request by DeityLink**
That's an idea I've had for a loooong time.

disk_duplicatorimage

:cl:
* rscadd: Added a new machine, the Data Disk Duplicator. Insert first a data disk of any type in the source slot, then insert a blank or corrupted disk in the target slot, and after a few second, the target disk will pop out as a carbon copy of the source disk, including the various datas that were on it. You can print its circuit board at RnD but it's also simple enough that you can also use a solding iron on a blank circuit board to craft it. Warranty void if emagged.
* rscadd: You can now print blank data disks at an autolathe under Misc Items. Diskette Boxes bought on the Merch computer already also come with a random amount of corrupted disks that finally have a purpose.

### CODE REVIEW

**Comment by DamianX** [comment]
I can't believe this has no interaction with the nuke disk

**Comment by gurfan** [comment]
finally, dupable nuke disks

**Comment by DeityLink** [comment]
dupable nuke disks is a fully intended mechanic and I can't wait to see the gameplay ramifications of it :^)

**Comment by Blithering** [comment]
I cannot wait for the first nuke round where sec secure the disk but find out too late the clown had made 100 copies and handed them out.

File: code/defines/obj/weapon.dm [hunk 1]
```diff
@@ -104,14 +104,18 @@
 
 /obj/item/weapon/disk
 	name = "Corrupted Data Disk"
-	desc = "The data on this disk has decayed, and cannot be read by any computer anymore."
+	desc = "The data on this disk has decayed, and cannot be read by any computer anymore. Can still be used by duplicator as if it were a blank disk."
```
Comment (Eneocho) [inline code/defines/obj/weapon.dm]:
```suggestion
	desc = "The data on this disk has decayed, and cannot be read by any computer anymore. Can still be used by a duplicator as if it were a blank disk."
```

**Comment by beesil** [comment]
very cool mechanics but I'm concerned that the sprite blends into basic metal tables. Could the base color be shifted up or down easily?

**Comment by DeityLink** [comment]
Probably, I'll tweak it tomorrow

**Comment by DeityLink** [comment]
There 
image
image
image
Similar color to the diskette boxes

**Comment by Eneocho** [comment]
Do the duplicate nuke disks have that respawning upon deletion thing too?

**Pull request merged**